hw5-10_to_do_list/react_ts/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "hw5-10_to_do_list",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.ts",
+  "scripts": {
+    "app": "node_modules/.bin/webpack serve & json-server --watch database/db.json",
+    "build": "node_modules/.bin/webpack",
+    "clear-dist": "rm -rf ./dist"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "devDependencies": {
+    "@types/react-dom": "^18.2.1",
+    "css-loader": "^6.7.3",
+    "html-loader": "^4.2.0",
+    "html-webpack-plugin": "^5.5.1",
+    "style-loader": "^3.3.2",
+    "ts-loader": "^9.4.2",
+    "typescript": "^5.0.4",
+    "webpack": "^5.80.0",
+    "webpack-cli": "^5.0.2",
+    "webpack-dev-server": "^4.13.3"
+  },
+  "dependencies": {
+    "@types/react": "^18.2.0",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0",
+    "prettier": "^2.8.8"
```

Comment:
to devDependencies
devDependencies - all code you don't expect to see in the final bundle

Reply:
fixed